lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -4954,7 +4954,9 @@ private void createEditTitleDialog(UIContainer tofill, SimplePage page, SimplePa
 				Collections.sort(siteMemberList, new Comparator<Member>() {
 					public int compare(Member lhs, Member rhs) {
 						UserSortNameComparator userComparator = new UserSortNameComparator();
-						return userComparator.compare(getUser(lhs.getUserId()), getUser(rhs.getUserId()));
+						Optional<User> lhsUserOptional = getUser(lhs.getUserId());
+						Optional<User> rhsUserOptional = getUser(rhs.getUserId());
+						return lhsUserOptional.isPresent() && rhsUserOptional.isPresent() ? userComparator.compare(lhsUserOptional.get(), rhsUserOptional.get()) : -1;
```

Comment:
Shouldn't you return 0 if either of the optionals are not present? It seems more neutral to return a match of zero.

Reply:
Ok